### DISCUSSION TITLE
Duda: ¿donde subir las prácticas?

### DISCUSSION THREAD

**Discussion by Andrelml** [Launch-X-Latam/MisionFrontEnd#187] [Q&A]
Hola a todos, mi duda es referente a la donde vamos a estar subiendo las prácticas, no se si tengamos que hacer un proyecto y ahí irlas almacenando o en un fork o en teams? 😳 ayuda please 🙏🏻🙏🏻

**Comment by Amhedriel** [comment]
Buenas, de momento no dijo donde subirlas o enviarlas, pero te recomiendo que los tengas listos en un repositorio tuyo, para cuando nos pidan solo lo envíes, es lo que yo trato de hacer [Amhedriel](https://github.com/Amhedriel/LaunchXPracticas "Prácticas en GitHub")

**Reply by Andrelml** [reply]
Muchas gracias amigo.

**Comment by Fernandoguate** [comment]
yo he tenido la misma duda, ya que de momento lo que ha pasado en la parte del boceto, wireframe y lo de figma pues no tengo idea de como subirlo, soy nuevo en esto y solo he subido proyectos de código en github.

**Reply by Amhedriel** [reply]
# ***Buenas Fernando***

Lo que puedes hacer es ver en esas páginas si tienen un botón o la opción de compartir lo que hiciste mediante un link.

**Me explico**: en mi repositorio [LaunchXPracticas](https://github.com/Amhedriel/LaunchXPracticas/blob/master/Practica1/Requerimientos/README.md "GitHub de Amhedriel") verás que lo comparto con un link, \[LaunchXPracticas](https://github.com/Amhedriel/LaunchXPracticas/blob/master/Practica1/Requerimientos/README.md "GitHub de Amhedriel"), hice una planilla de [Fases de Requerimientos](https://1drv.ms/p/s!AoBVO6v6zSGpgfwLtOUPeXhrGYcd-g?e=h9lTDA "Power Point") en Power Point y estoy a medio [Wireframe de Abogabot](https://wireframe.cc/1p4Qrj "Wireframe.cc") pero estos últimos los comparto con los links de internet, no sé si tienen permisos para modificarlos, pero confío en que no lo hagan =P.

---

Ahora bien, si alguien que lee esto se pregunta cómo se hace parte de lo que escribí, les dejo un tutorial de MarkDown, que es utilizado en GitHub y otros lugares para poder documentar sus proyectos, sería un equivalente a usar HTML y ayuda a comprender algunas cosas, pero recomiendo primero aprender HTML y luego saltar a [MarkDown](https://github.com/Amhedriel/Mark_Down_Tutoriales/blob/master/MarkdownRapido.md) en este caso es un enlace a uno de mis repositorios que tiene un resumen de MD, para ver la sintaxis de ese repositorio, clónalo y míralo en tu IDE favorito.

Quieres ver algunos tutoriales en construcción de Bash y Git? [Siguiente 🠒](https://github.com/Amhedriel/Mark_Down_Tutoriales/blob/master/README.md)

Saludos.

**Comment by fabiancgc12** [comment]
Tengo entendido que mas adelante abrirán un formulario donde podrás enviar tus practicas. Igual lo que puedes ir haciendo es abrir un repositorio para cada practica y allí vas subiendo lo que tengas
